src/vm/i386/excepx86.cpp

```diff
@@ -3769,10 +3783,27 @@ AdjustContextForVirtualStub(
     return TRUE;
 }
 
+#ifndef DACCESS_COMPILE
+#ifndef WIN64EXCEPTIONS
 #ifdef FEATURE_PAL
+EXTERN_C
+_Unwind_Reason_Code
+UnhandledExceptionHandlerUnix(
```

Comment:
I am not sure I understand. The function already has implementation in exceptionhadling.cpp, so this would go away completely.